modules/negotiate.go

```diff
@@ -21,44 +21,26 @@ const (
 	// termination, i.e. that the sender wishes to cease communication, but
 	// not due to an error.
 	StopResponse = "stop"
+)
 
+const (
```

Comment:
why the extra `const` block here? Just separating the `Responses` from the other constants?

Reply:
Yes, just to fix `sorted_consts`'s warning.